tests/integration/ceph_multi_cluster_test.go

```diff
@@ -58,20 +58,22 @@ func TestCephMultiClusterDeploySuite(t *testing.T) {
 
 type MultiClusterDeploySuite struct {
 	suite.Suite
-	testClient *clients.TestClient
-	k8sh       *utils.K8sHelper
-	namespace1 string
-	namespace2 string
-	op         *MCTestOperations
+	testClient       *clients.TestClient
+	k8sh             *utils.K8sHelper
+	namespace1       string
+	namespace2       string
+	testPVCNamespace string
+	testPVC          bool
+	op               *MCTestOperations
 }
 
 // Deploy Multiple Rook clusters
 func (mrc *MultiClusterDeploySuite) SetupSuite() {
 
 	mrc.namespace1 = "mrc-n1"
 	mrc.namespace2 = "mrc-n2"
-
-	mrc.op, mrc.k8sh = NewMCTestOperations(mrc.T, mrc.namespace1, mrc.namespace2)
+	mrc.testPVCNamespace = "mrc-n3"
```

Comment:
Why do we need a third namespace? If we're going to test on PVCs, I thought we could just use namespace1 and namespace2 for those new OSDs. Let's just convert this whole test to a PVC test as long as conditions are met (1.13+ or other conditions). If the conditions are not met, we will not use PVCs. Everything else besides OSDs in the test can remain the same.

Reply:
I was thinking not to change the existing tests of hostPathDir instead of that we can test the osd over PVC on a separate namespace(mrc-n3). So what happens currently is if the `gp2` storageClass is present it will create a new namespace `mrc-n3` and and deploy a third ceph cluster(having osds deployed over PVCs) there and test it.

Reply:
Per a comment I linked previously, we eventually won't be able to have a multi-cluster test that runs on devices, or else we will need to worry about complications of the clusters using the same devices. It will be simplest if this multi-cluster test just uses PVCs. The other test suites will already cover the scenario of validating without PVCs, so no need for this suite to use both types of OSDs.